docker/build_matrix.yaml

```diff
@@ -95,3 +95,55 @@
   - mosaicml/pytorch_vision:latest_cpu
   TARGET: vision_stage
   TORCHVISION_VERSION: 0.12.0
+- BASE_IMAGE: nvidia/cuda:11.3.1-cudnn8-devel-ubuntu20.04
```

Comment:
In the docs it says the base image is `mosaicml/pytorch` but here the base images are all `nvidia/cuda` - how come?

Reply:
We use staged builds, so the BASE_IMAGE arg is always the root base image, but the Dockerfile code path effectively builds it on top of mosaicml/pytorch_vision.